src/components/app.js

```diff
@@ -24,12 +27,20 @@ class App extends Component {
     });
   };
 
+  handleSearch = (e) => {
+    this.setState({search: e.target.value});
+  };
+
+  filterList = (list, value) => list.filter(item => item.name.includes(value));
```

Comment:
If you want it as a class member the argument is not needed since you can access state via this

Reply:
Moved to utils